docs/core/preview3/tools/dotnet-add-package.md

```diff
@@ -17,20 +17,20 @@ ms.assetid: 88e0da69-a5ea-46cc-8b46-5493242b7af9
 
 ## Name
 
-`dotnet-add package`
+`dotnet-add package` - Adds a package reference to a project file.
 
 ## Synopsis
 
 ```
-dotnet add [<PROJECT>] package <PACKAGE_NAME> [-v|--version] [-f|--framework] [-s|--source] [--package-directory] [-h|--help]
+dotnet add [<PROJECT>] package <PACKAGE_NAME> [-v|--version] [-f|--framework] [-s|--source] [--package-directory]
```

Comment:
I know it's possible, but would you recommend people to use --help with any other option? The result will always be the same as `dotnet add package -h`. Also, the first line is only good for saying you can do `dotnet add package packagename [other options] --help`, but it doesn't cover the `dotnet add -h` case, which is the parent command.